### DISCUSSION TITLE
Export Nemo non-English models into Jarvis

### DISCUSSION THREAD

**Discussion by santosh-avaya** [NVIDIA-NeMo/NeMo#2429] [Q&A] [answered]
We have a few queries wrt ASR. Appreciated if someone can answer:
- We would like to export non-English (.nemo) models into Jarvis (.jmir and then deploy) and do production grade real-time inferencing. Is that possible seamlessly ? Should we follow TLT pipeline ?
- Is it possible to load multiple ASR models in single Jarvis instance ?
- jarvis_asr_proto file has configuration paraemeter as RecognitionConfig.model = 13. Is it possible to change the value without restarting the container ? What are possible values of this parameter ?
- jarvis_asr_proto file has configuration paraemeter as RecognitionConfig.language_code = 3. Is it possible to change the value without restrarting the container ? What are possible values of this parameter ?

Thanks,
Santosh

**Comment by ryanleary** [comment, marked as answer]
Hi Santosh,

1. If you already have non-English nemo models, you can use the nemo2jarvis tool released in Jarvis 1.3.0 to convert to `ejrvs` then use the jarvis-build/jarvis-deploy tooling.
2. Yes, you can load multiple ASR models in single Jarvis instance. When using jarvis-build, you will want to give the pipelines unique names and set the language flag appropriately for the model.
3. You cannot change the values in the proto files. The proto files are for generating client library code. Once the code is generated, the `model` and `language_code` fields are strings. You can then request a specific model (matching the name you configured with `jarvis-build`) and/or language.

**Reply by titu1994** [reply]
Adding to this, this discussion forum is primarily for Nemo only. For all discussion involving Riva, including nemo2riva, please ask on Rivas discussion forum here - https://forums.developer.nvidia.com/c/ai-data-science/deep-learning/riva/475
